cmd/erasure-healing.go

```diff
@@ -1133,6 +1162,35 @@ func isObjectDangling(metaArr []FileInfo, errs []error, dataErrs []error) (valid
 	return validMeta, totalErrs > validMeta.Erasure.ParityBlocks
 }
 
+// Bucket is considered dangling/corrupted if and only
+// if bucket meta data is missing on more than N/2 disks
+func isBucketDangling(metaArr []VolInfo, errs []error) bool {
+	// We can consider a bucket not reliable
+	// when xl.meta is not found in read quorum disks.
+	// or when xl.meta is not readable in read quorum disks.
+	danglingErrsCount := func(cerrs []error) (int, int, int) {
+		var (
+			notFoundCount     int
+			corruptedCount    int
+			diskNotFoundCount int
+		)
+		for _, readErr := range cerrs {
+			switch {
+			case errors.Is(readErr, errVolumeNotFound):
+				notFoundCount++
+			case errors.Is(readErr, errFileCorrupt):
```

Comment:
No file corrupt error in my opinion is returned in this case

Reply:
ack. will remove